src/shared/common/components/workstack.jsx

```diff
@@ -92,7 +92,7 @@ class WorkstackAllocate extends Component {
         const { selectedCases } = this.state;
         if (filter !== '') {
             const filterableColumns = columns.filter(column => {
-                return column.isFilterable === true;
+                return column.filterable === true;
```

Comment:
Same as above, i suspect once we fix the returning objects in info service. This part can be reverted back to 'isFilterable'.